server/routes/contacts/manage/domestic-status/manageDomesticStatusController.test.ts

```diff
@@ -0,0 +1,72 @@
+import type { Express } from 'express'
+import request from 'supertest'
+import { appWithAllRoutes, user } from '../../../testutils/appSetup'
+import AuditService, { Page } from '../../../../services/auditService'
+import PrisonerSearchService from '../../../../services/prisonerSearchService'
+import ContactsService from '../../../../services/contactsService'
+import ReferenceDataService from '../../../../services/referenceDataService'
+import TestData from '../../../testutils/testData'
+import { mockedReferenceData } from '../../../testutils/stubReferenceData'
+
+jest.mock('../../../../services/auditService')
+jest.mock('../../../../services/prisonerSearchService')
+jest.mock('../../../../services/contactsService')
+jest.mock('../../../../services/referenceDataService')
+
+const auditService = new AuditService(null) as jest.Mocked<AuditService>
+const prisonerSearchService = new PrisonerSearchService(null) as jest.Mocked<PrisonerSearchService>
+const contactsService = new ContactsService(null) as jest.Mocked<ContactsService>
+const referenceDataService = new ReferenceDataService(null) as jest.Mocked<ReferenceDataService>
+
+let app: Express
+const prisonerNumber = 'A1234BC'
+const contactId = '10'
+
+beforeEach(() => {
+  app = appWithAllRoutes({
+    services: {
+      auditService,
+      prisonerSearchService,
+      contactsService,
+      referenceDataService,
+    },
+    userSupplier: () => user,
+  })
+  referenceDataService.getReferenceData.mockImplementation(mockedReferenceData)
+})
+
+afterEach(() => {
+  jest.resetAllMocks()
+})
+
+describe('GET /prisoner/:prisonerNumber/contacts/manage/:contactId/domestic-status', () => {
+  it('should render manage domestic status page', async () => {
+    // Given
+    auditService.logPageView.mockResolvedValue(null)
+    prisonerSearchService.getByPrisonerNumber.mockResolvedValue(TestData.prisoner())
+    contactsService.getContact.mockResolvedValue(TestData.contact())
+
+    // When
+    const response = await request(app).get(`/prisoner/${prisonerNumber}/contacts/manage/${contactId}/domestic-status`)
+
+    // Then
+    expect(response.status).toEqual(200)
+    expect(auditService.logPageView).toHaveBeenCalledWith(Page.MANAGE_DOMESTIC_STATUS_PAGE, {
```

Comment:
this should check if the field is set with correct value?

Reply:
Unit test added for status selected